.github/chainguard/scrape-error-comments.sts.yaml

```diff
@@ -1,7 +1,8 @@
 issuer: https://accounts.google.com
 
-# detect@jrawlings-chainguard.iam.gserviceaccount.com
-subject: "114294174394359664841"
+# 114294174394359664841: detect@jrawlings-chainguard.iam.gserviceaccount.com
```

Comment:
Do we still need this?

Reply:
Possibly, if something doesn't work out and we need to rework some logic.  It's hard to mock all the comments we've had so this is the best test bed to pull from.  For early dev being able to use a dev cloud run env has been helpful.